libraries/AP_AHRS/AP_AHRS_DCM.cpp

```diff
@@ -116,6 +116,9 @@ AP_AHRS_DCM::update(bool skip_ins_update)
     update_AOA_SSA();
 
     backup_attitude();
+
+    // update takeoff/touchdown flags
+    update_flags();
```

Comment:
I find it a bit sad that the general purpose AHRS::update() is in AP_AHRS_DCM.  This is correct though right, this is the only place we can put it?

Reply:
we could refactor it, but with our current structure that is the logical place